app/src/main/res/drawable-sw600dp-land/onboarding_dot_active.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="oval">
+  <solid android:color="@color/oppiaPrimaryText" />
+  <corners android:radius="5dp" />
+</shape>
```

Comment:
Remove this file and use the existing `onboarding_dot_active`.
We normally round up the dimensions to nearest dimension `4dp` and as it is `5px` in mocks we can use `4dp` here.

All dimensions should be in multiple of `4dp`. (There are some exceptions like border strokes)

Reply:
Done